projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -245,6 +245,25 @@ export class IgxOverlayService implements OnDestroy {
             false);
     }
 
+    /**
+     * Repositions the component in the horizontal and/or vertical directions.
+     * ```typescript
+     * this.overlay.setOffset(id, x, y);
+     * ```
+     */
+    setOffset(id: string, deltaX: number, deltaY: number) {
+        const info: OverlayInfo = this.getOverlayById(id);
+
+        if (!info) {
+           return;
+        }
+
+        info.transformX += deltaX;
+        info.transformY += deltaY;
+
+        info.elementRef.nativeElement.parentElement.style.transform = 'translate(' + info.transformX + 'px, ' + info.transformY + 'px)';
```

Comment:
use [string interpolation](https://basarat.gitbooks.io/typescript/docs/template-strings.html) here - it is much more beautiful 😄